config/jobs/ppc64le-cloud/build-docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,166 @@
+periodics:
+    - name: periodic-config-docker
+      cluster: k8s-ppc64le-cluster
+      cron: 0 0 * * *
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+          slack:
+            channel: 'prow-job-notifications'
+            report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:20890c8770ff57f91eb9f16fbb9f9e979958cd02b2a3c03795d2587e7b58a9db
+          resources:
+            requests:
+              cpu: "8000m"
+            limits:
+              cpu: "8000m"
+          command:
+          - /bin/bash
+          args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+            echo "* Start prow-info-docker *"
+            chmod ug+x $PWD/upstream-master-ci/prow-info-docker.sh
+            $PWD/upstream-master-ci/prow-info-docker.sh
+            rc=$?
+            [ $rc != 0 ] && echo "ERROR: prow-info-docker exited with code:$rc"
+            exit $rc
+          securityContext:
+            privileged: true
```

Comment:
Is the below required for all four jobs?
```
securityContext:
  privileged: true
```

Reply:
Yes, `dockerd` refuses to start otherwise.
`failed to start daemon: Error initializing network controller: error obtaining controller instance: failed to create NAT chain DOCKER: iptables failed: iptables -t nat -N DOCKER: iptables v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)`